vcpkg/users/triplets.md

```diff
@@ -161,6 +161,17 @@ set(VCPKG_MAKE_CONFIGURE_OPTIONS "lt_cv_deplibs_check_method=pass_all")
 
 Also available as build-type specific `VCPKG_MAKE_CONFIGURE_OPTIONS_DEBUG` and `VCPKG_MAKE_CONFIGURE_OPTIONS_RELEASE` variables.
 
+### VCPKG_HASH_ADDITIONAL_FILES
+
+A cmake list of files to include into the calculation of the abi hash. This is for example required if files defined in `VCPKG_MESON_(NATIVE|CROSS)_FILE(_RELEASE|_DEBUG)?`
+should be part of the calculated abi or if files are included in the triplet/toolchain provided to vcpkg and those files should take part in the abi hash.
```

Comment:
Are relative paths supported? What are they relative to?

Reply:
> Are relative paths supported? What are they relative to?

Theoretical yes. Same behavior as `_wfopen_s` since that is what `get_file_hash` seems to boil down to. So it is relative to `CWD` I would assume. However using absolute paths via `CMAKE_CURRENT_LIST_DIR` is consistent in regards on how `VCPKG_CHAINLOAD_TOOLCHHAIN_FILE` needs to be defined.